keanu-project/src/main/java/io/improbable/keanu/tensor/generic/GenericTensor.java

```diff
@@ -73,6 +75,35 @@ public T getValue(int... index) {
         return this;
     }
 
+    public boolean isNull() {
+        return this.data == null;
+    }
+
+    public GenericTensor<T> setWithMaskInPlace(DoubleTensor mask, T value) {
```

Comment:
Awesome that we have a new util for this! Should this be generalised across all tensors?